### PR TITLE
doc: README Rework

### PR DESCRIPTION
**Pull request by LennartPurucker**
Make the README better!

### CODE REVIEW

**Comment by LennartPurucker** [comment]
I suggest looking at the README in the branch instead of the PR diff.

**Review by PGijsbers — CHANGES_REQUESTED** [review]
Wow Lennart 😱 that's amazing 🔥 our stars ⭐  are going 📈 !
But why do we stop at 12? 🤔  We can go all the way to 13! 🚀

File: README.md [hunk 1]
```diff
@@ -1,22 +1,76 @@
...
+It can download or upload data from OpenML, such as datasets and machine learning experiment results.
+
+## :joystick: Minimal Example
+
+Use the following code to get the [credit-g](https://www.openml.org/search?type=data&sort=runs&status=active&id=31) dataset for machine learning:
+
+```python
+import openml
+
+dataset = openml.datasets.get_dataset(31)
```
[the remark this answers is not in the document]
Comment (eddiebergman) [reply README.md:36]: my friend, credit-g, always in our hearts

File: README.md [hunk 2]
```diff
@@ -1,22 +1,76 @@
...
+[![Latest Release](https://img.shields.io/github/v/release/openml/openml-python)](https://github.com/openml/openml-python/releases)
+[![Python Versions](https://img.shields.io/badge/python-3.8%20%7C%203.9%20%7C%203.10%20%7C%203.11%20%7C%203.12-blue)](https://pypi.org/project/openml/)
+[![Downloads](https://static.pepy.tech/badge/openml)](https://pepy.tech/project/openml)
 [![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
 
-## Citing OpenML-Python
 
-If you use OpenML-Python in a scientific publication, we would appreciate a reference to the
-following paper:
+[Installation](https://openml.github.io/openml-python/main/#how-to-get-openml-for-python) | [Documentation](https://openml.github.io/openml-python) | [Contribution guidelines](https://github.com/openml/openml-python/blob/develop/CONTRIBUTING.md)
```
[the remark this answers is not in the document]
Comment (LennartPurucker) [reply README.md:23]: I would rather want to point to the github md

**Review by PGijsbers — APPROVED** [review]
🥇

**Pull request merged**